src/main/menu.ts

```diff
@@ -72,6 +75,12 @@ export class MenuBuilder {
             this.onOpenPreferences();
           },
         },
+        {
+          label: "Open Application Support Folder",
```

Comment:
This is looking pretty good but I think it's still not very clear what this should do for the end user. Is this the `Application Support` folder for the launcher? Is it for the `Application Support` folder for Dolphin? If it's for Dolphin, which Dolphin? Netplay or Playback? I just don't think it's very clear what this should do and when it should be used.

Reply:
You might consider just "Open Slippi Dolphin Application Support Folder" or something, since Slippi Dolphin shares a common folder with subdirectories for netplay and playback-specific stuff.

Reply:
Ah, question then: am I misunderstanding what you want to solve here?

e.g, user currently digs into Slippi Dolphin bundle to do x/y/z, that would now need to dig into `~/Library/Application\ Support/com.project-slippi.dolphin/` - this, as it stands, would be opening `~/Library/Application\ Support/Slippi\ Launcher/...`, no? This is assuming this change is combined with having Dolphin extract the folder on-first-run.

(Or am I conflating multiple unrelated things here?)